packages/mongo-nav/src/project-nav/ProjectNav.tsx

```diff
@@ -248,9 +246,9 @@ type ProjectNavProps = Pick<
   onPremVersion?: string;
   onPremMFA?: boolean;
   alerts?: number;
-  urls: Required<URLSInterface>;
-  hosts: Required<HostsInterface>;
-  constructProjectURL: (orgID: string, projectID: string) => string;
+  constructProjectURL: NonNullable<MongoNavInterface['constructProjectURL']>;
+  urls: Required<NonNullable<MongoNavInterface['urls']>>;
```

Comment:
For my own knowledge, do we need both required and nonnullable?

Reply:
It's what made TS happy. I think cuz the object itself is marked Required I had to add this.